controllers/articles-controllers.js

```diff
@@ -29,23 +35,23 @@ const {
           
           res.status(200).send( {results });
           
-        }) .catch((err) => {
-          next(err);
-        });
+        }) 
+        // .catch((err) => {
+        //   next(err);
+        // });
         }
         exports.getCommentsByArticle = (req, res, next) => {
         const id =req.params.article_id;
         
         
           selectCommentsByArticle(id).then((results) => {
            
-            if (results.length===0) {res.status(404).send({ msg:'article doesn\'t exist' });
-          }else{
+            
             res.status(200).send( {results });
-          }
+          
             
           }) .catch((error) => {
-            res.status(500).send({ msg: 'An error occurred' });
-  });
-          };
-          
\ No newline at end of file
+            const{status, msg}=error
```

Comment:
Just need to pass error to next in the error handling middleware chain using ```next``` , we don't do the res.status.send for the error here as this is handled by the error handling middleware